changelog/8498.removal.md

```diff
@@ -0,0 +1 @@
+Update MINIMUM_COMPATIBLE_VERSION to "2.6.0".
```

Comment:
Can we please phrase that more user-facing? Users should know that this means that they have to retrain the model with 2.6 in order to run with 2.6